lib/goals.js

```diff
@@ -207,13 +207,13 @@ class GoalLookAtBlock extends Goal {
 // Path into a position were a blockface of block at x y z is visible.
 // You'll manually need to break the block. THIS WONT BREAK IT
 class GoalBreakBlock extends Goal {
-  constructor (x, y, z, bot, options = {}) {
+  constructor (pos, bot, options = {}) {
```

Comment:
Yeah but why? Documentation in readme.md says `GoalBreakBlock(x, y, z, bot, options)`. I can't merge it without a new major release as it would be a breaking API change.